components/NewPersonView/PersonHistory.tsx

```diff
@@ -31,9 +37,16 @@ const PersonHistory = ({ personId }: Props): React.ReactElement => {
     return <ErrorMessage label={casesError.message} />;
   }
 
+  let deletedCount = 0;
+
   const events = [] as Case[];
   for (let i = 0; casesData !== undefined && i < casesData.length; i++) {
     events.push(...casesData[i].cases);
+    if (includeDeletedCount) {
+      deletedCount = casesData[i].deletedCount
+        ? Number(casesData[i].deletedCount)
```

Comment:
I think based on [this PR](https://github.com/LBHackney-IT/social-care-case-viewer-api/pull/538/files) the `deletedCount` will be returned as `DeletedRecordsCount` from the backend

Reply:
updated!